src/booleans.js

```diff
@@ -1,62 +1,109 @@
 function negate(a) {
-  // your code here
-};
+  if (a === false) {
+    return true;
+  }
+  return false;
+}
 
 function both(a, b) {
-  // your code here
-};
+  if (a === true && b === true) {
+    return true;
+  }
+  return false;
```

Comment:
This is a long way of doing this, though it does work! It can be boiled down to just:
```
  return a && b;
```